docs/guides/admin/docs/configuration/stream-security.md

```diff
@@ -228,7 +229,7 @@ is working as expected. It is especially important to try to access resources th
 
 There are ways to test in a structured way which will be explained below.
 
-### Creating Signed URLs with Signing Endpoint
+### Test 1: Creating Signed URLs with Signing Endpoint
```

Comment:
This is one test with multiple steps, not multiple tests. Naming it test 1, 2, … seems odd.

Reply:
Ah, in line 230 above it, it reads `There are ways to test` followed by a list of multiple items. 

If it is only the one test with multiple steps, I would change the Test 1, 2, etc. to Step 1, 2, etc. and change the aforementioned sentence to read something like `The following explains how to test if Stream Security has been correctly configured.`